modules/Bio/EnsEMBL/Variation/Pipeline/PhenotypeAnnotation/BasePhenotypeAnnotation.pm

```diff
@@ -689,15 +689,21 @@ sub dump_phenotypes {
   };
 
   my $db_adaptor    = $self->variation_db_adaptor;
+  
 
-  _sql_to_file($pfa_select_stmt, $db_adaptor, $self->workdir."/"."pfa_".$source_name.".txt");
-  _sql_to_file($pf_select_stmt, $db_adaptor, $self->workdir."/"."pf_".$source_name.".txt");
-  _sql_to_file($p_extra_select_stmt, $db_adaptor, $self->workdir."/"."p_extra_".$source_name.".txt");
-  _sql_to_file($poa_extra_select_stmt, $db_adaptor, $self->workdir."/"."poa_extra_".$source_name.".txt");
+  mkdir $self->workdir."/previous_data", 0755;
```

Comment:
Instead of `mkdir` you could use `make_path`.
See here: 
https://github.com/Ensembl/ensembl-variation/blob/main/modules/Bio/EnsEMBL/Variation/Pipeline/PhenotypeAnnotation/ImportMouse.pm#L43
https://github.com/Ensembl/ensembl-variation/blob/main/modules/Bio/EnsEMBL/Variation/Pipeline/PhenotypeAnnotation/ImportMouse.pm#L68-L69